course3/homework-unique_ptr/my_unique_ptr.h

```diff
@@ -0,0 +1,93 @@
+#include <cstddef>
+#include <iostream>
+
+template<class T>
+class MyUniquePtr
+{
+public:
+	MyUniquePtr() : mData(nullptr)
+	{
+		std::cout << "MyUniquePtr::ctor" << std::endl;
+	}
+	
+	MyUniquePtr(MyUniquePtr& that)
```

Comment:
Nope. This isn't the move constructor. This constructor can steal ownership from an lvalue object. Move constructor takes rvalue pointer.